src/components/forms/QuestionCreationForm.js

```diff
@@ -84,15 +88,24 @@ const QuestionCreationForm = ({
     </div>
 
     <div className="questionInput questionContent">
-      <Field name="content" component={SCCreationContent} />
+      <Field
+        name="content"
+        component={type === 'SC' || type === 'MC' ? SCCreationContent : SCCreationContent}
+      />
     </div>
 
-    <div className="questionInput questionOptions">
-      <Field name="options" component={SCCreationOptions} />
-    </div>
+    {type !== 'FREE' && (
```

Comment:
`<div className="questionInput....` should wrap the different options (don't contain inside {type !== ...}